README.md

```diff
@@ -11,20 +11,21 @@ A simple Teamspeak stat-generator - based on server-logs
 # Usage
 - Run the script `tsstats [-h]`
 - Optionally create a config-file (see [Configuration](https://github.com/Thor77/TeamspeakStats#configuration))
+- The package works entirely off your Teamspeak server's logs, so that no ServerQuery account is necessary
 
 # Example
 ```
 tsstats -l /var/log/teamspeak3-server/ts3server*.log -o /var/www/tsstats.html
 ```
 
-Parse logs matching `ts3server*.log` in `/var/log/teamspeak3-server` and write output to `/var/www/tsstats.html`
+Parse logs matching `ts3server*.log` in `/var/log/teamspeak3-server` and write output to `/var/www/tsstats.html`.
 
 # CLI-Usage
 ```
 usage: tsstats [-h] [-c CONFIG] [--idmap IDMAP] [-l LOG] [-o OUTPUT] [-d]
                    [-nod]
 
-A simple Teamspeak stats-generator - based on server-logs
+A simple Teamspeak stats-generator - based solely on server-logs
```

Comment:
Please change that [here](https://github.com/Thor77/TeamspeakStats/blob/master/tsstats/__main__.py#L18) instead, as I would accidentially overwrite it this way, once I add smth new to the cli (as that's just the output of `tsstats -h`).